src/wallet/rpcwallet.cpp

```diff
@@ -1403,6 +1403,16 @@ UniValue ListReceived(CWallet * const pwallet, const UniValue& params, bool fByA
         if(params[2].get_bool())
             filter = filter | ISMINE_WATCH_ONLY;
 
+    bool fFilterAddress = false;
```

Comment:
The code convention has changed a bit recently. Now, we don't put the type as prefix, and we try to use snake case. I would rename this `has_filter_address` and the below to `filter_address`. You can leave existing vars as is, if you don't want the diff to grow for no reason though (like `fByAccounts` below).